### PR TITLE
add blog post about the atoum day feedback

### PR DESCRIPTION
**Pull request by Grummfy**
WIP, but I would like to have a first feedback on it.
thanks

ps : writen in the train with screaming little girl ...

### CODE REVIEW

**Comment by Grummfy** [comment]
@mikaelrandy @jubianchi @agallou @Hywan un avis?

File: _posts/2016-06-17-back-from-atoum-day.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+date: 2016-06-17 12:30:00
+author: "@Grummfy"
+categories: event
+---
+
+# Back from atoum's day 2016
+
+This year we got the first [2016-04-27-atoum-s-day](atoum's day)! This event is both a meeting between contributors and a meeting between users.
+
+Unfortunatelly, due to some strike in France during this event, every registred people couldn't reach us and some need to leave earlier. Hopefully, it still was a good meeting, with intresting talks. We hope, as team, that our few users that were there enjoy the moment too.
```
Comment (agallou) [inline _posts/2016-06-17-back-from-atoum-day.md]: intresting -> interesting ?

File: _posts/2016-06-17-back-from-atoum-day.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+But this event, was full of surprise and quite exciting! We are really happy to speak about it, and we will probably create a new event of this kind in the future.
+
+## Team meeting
+
+We create this event also to speak between each contributors. This allow us to reach some points, and think to a new roadmap. We also check what we could do to improve atoum and his community. Some of the point will be detailled lower and some will be explain during the year. We would like to keep some new stuff under the hood, to allow you discovering it.
```
Comment (agallou) [inline _posts/2016-06-17-back-from-atoum-day.md]: create ? mettre au passé ?

Comment (agallou) [inline _posts/2016-06-17-back-from-atoum-day.md]: lay a new roadmap ?

Comment (agallou) [inline _posts/2016-06-17-back-from-atoum-day.md]: check : passé ?

File: _posts/2016-06-17-back-from-atoum-day.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+One of the point we speak about was the [http://event.afup.org/php-tour-2016/](phptour) of this year. During this event we got an area in the stand of [http://www.norsys.fr/](Norsys). It help us to talk to people, make some demo, and give our first goodies (some stickers) to you, our community. And we enjoy it ;)
+
+## Talks and user meeting
+
+We got four talks, instead of six due to time and strike. But it was very intresting talks and each of them were registred (badly, but we got it). You can discover it on youtube, if you understand french (or the google subtitle), in the (https://www.youtube.com/playlist?list=PL_KNbXDBJ__SFuqpr-_g3elWJzctoLpwA)[dedicated playlist].
+
+
+## Roadmap & next steps
+
+We try to think to what are missing now or what was blaoted in atoum and try to create the next steps. For the code, we still need to think about the points we identified, before spreading them. By the way, if you want to know more about it, just join us in our (https://webchat.freenode.net/?channels=##atoum)[chat room] and ask us.
```
Comment (agallou) [inline _posts/2016-06-17-back-from-atoum-day.md]: première phrase peu comprehensible.

File: _posts/2016-06-17-back-from-atoum-day.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+About the rest, we almost put in evidence that our main weakness is the communication. So we will try to spread more words about atoum on social media but also in other place like meetup or conferences.
+
+Some improvement that we see are sometimes just quick-win or nearly quick-win like
+
+* blog comments, that are now available
+* a "standard edition" of atoum with some extension build-in. A preview is available (https://github.com/agallou/atoum-standard-edition)[there].
```
Comment (agallou) [inline _posts/2016-06-17-back-from-atoum-day.md]: faut-il mettre un lien de suite. Peut être expliquer rapidement le concept et indiquer. "more to come about this soon in a next post" ?

File: _posts/2016-06-17-back-from-atoum-day.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+* create a more clear communication related to our roadmap
+
+Some points are more complicated but also more crucial and because we love you, here is one of this point. We know that security are important for you, we will start to use gpg key to sign our commit and mak it more trustable. We already know that we need to create a build machine for our phar, but once it will be created, we will sign our phar too. This will allow you to check the validity of the new phar and improve your security.
+
+And we still got some other things under the hood... Like we say at the start of this post, this event was intresting and concluant!
+
+
+## Conclusion
+
+Sometimes, when you create a new project you get some secreat dream about it. Sometimes, you just want to create great tools for great people. And sometimes this comes to reality. When the [https://www.github.com/mageekguy](Frédéric Hardy) create ogo, and first answer to his needs. But after a while and renaming it to atoum, a dreams comes in. A dream about a meeting beetween contributors and users, a dream about a day with some kind of magic.
```
Comment (agallou) [inline _posts/2016-06-17-back-from-atoum-day.md]: secreat ?

File: _posts/2016-06-17-back-from-atoum-day.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+A few years later, after the the leaving of Frédéric, the new team create this day, without knowing it was a dream. This is the magic of dream.
```
Comment (agallou) [inline _posts/2016-06-17-back-from-atoum-day.md]: create -> temps

**Comment by Grummfy** [comment]
Here is the updated text. thanks @agallou . Because it's a branch of this repository, don't hesitate to change it.

File: _posts/2016-06-17-back-from-atoum-day.md (hunk 3 above)
Comment (Grummfy) [inline _posts/2016-06-17-back-from-atoum-day.md]: which one?

**Pull request merged**